### PR TITLE
Filetemplates for dart, fortran, javascript, lua, and rust

### PR DESCRIPTION
**Pull request by youknowwhoiam05**
I have added **starting templates** for lua, fortran, javascript , rust, and dart 😃

### CODE REVIEW

**Comment by Pushpavel** [comment]
Thanks, da. I noticed that a blank first character would be created in the rust template and  file name should follow Title case (Language name must be capitalized)

**Review by Pushpavel — CHANGES_REQUESTED** [review]
Rust Template would create a blank first character.Fix it

File: src/main/resources/fileTemplates/j2ee/Dart.dart.ft [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#**
+ * Dart language Competitive Programming Template
+ *
+ * wrap the code like #[[ code ]]# so that it is generated as it is
+ *#
+#[[main(){
```
Comment (Pushpavel) [inline src/main/resources/fileTemplates/j2ee/Dart.dart.ft:6]: can have a newline here

File: src/main/resources/fileTemplates/j2ee/Javascript.js.ft [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#**
+ * Javascript Competitive Programming Template
+ *
+ * wrap the code like #[[ code ]]# so that it is generated as it is
+ *#
+#[[]]#
```
Comment (Pushpavel) [inline src/main/resources/fileTemplates/j2ee/Javascript.js.ft:6]: have at least one line

File: src/main/resources/fileTemplates/j2ee/Lua.lua.ft [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#**
+ * lua Competitive Programming Template
+ *
+ * wrap the code like #[[ code ]]# so that it is generated as it is
+ *#
+#[[]]#
```
Comment (Pushpavel) [inline src/main/resources/fileTemplates/j2ee/Lua.lua.ft:6]: have at least one line

File: src/main/resources/fileTemplates/j2ee/Rust.rs.ft [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#**
+ * Rust Competitive Programming Template
+ *
+ * wrap the code like #[[ code ]]# so that it is generated as it is
+ *#
+#[[fn main() {
```
Comment (Pushpavel) [inline src/main/resources/fileTemplates/j2ee/Rust.rs.ft:6]: start and end this template with a new line

Comment (youknowwhoiam05) [reply src/main/resources/fileTemplates/j2ee/Rust.rs.ft]: done

File: src/main/resources/fileTemplates/j2ee/Lua.lua.ft (hunk 3 above)
Comment (youknowwhoiam05) [reply src/main/resources/fileTemplates/j2ee/Lua.lua.ft]: done

File: src/main/resources/fileTemplates/j2ee/Javascript.js.ft (hunk 2 above)
Comment (youknowwhoiam05) [reply src/main/resources/fileTemplates/j2ee/Javascript.js.ft]: done

File: src/main/resources/fileTemplates/j2ee/Dart.dart.ft (hunk 1 above)
Comment (youknowwhoiam05) [reply src/main/resources/fileTemplates/j2ee/Dart.dart.ft]: done :)

**Review by youknowwhoiam05 — COMMENTED** [review]
i guess all spacing are okay now

**Review by Pushpavel — APPROVED** [review]
Super 🔥

**Pull request merged**